backend/vxlan/utils.go

```diff
@@ -151,3 +151,50 @@ func getHostsMap(hosts []metadata.Host) map[string]metadata.Host {
 	logrus.Debugf("hostsMap: %v", hostsMap)
 	return hostsMap
 }
+
+func getLocalRouterInfo(networks []metadata.Network, host metadata.Host, services []metadata.Service) (metadata.Network, metadata.Container, metadata.Service) {
```

Comment:
We don't have the concept of router anymore. It would be confusing to use this word. Can you please change this to something else? Also might be more easy to read if we don't combine fetching network and self service?